packages/grafana-data/src/types/app.ts

```diff
@@ -17,7 +17,12 @@ export interface AppRootProps<T = KeyValue> {
   /**
    * Pass the nav model to the container... is there a better way?
    */
-  onNavChanged: (nav: NavModel) => void;
+  onNavChanged: (nav: NavModel | null) => void;
+
+  /**
+   * Pass a boolean to either show or hide the default styling that the parent App Wrapper has.
+   */
+  removeParentStyling?: (removeParentStyling: boolean) => void;
```

Comment:
rather than a boolean flag..  maybe we could have a enum?  To start with, it would just be two states... with and without header?

Reply:
Should this not be on the page level instead of the whole app plugin? And yes an enum like PageStyle { Centered | Full width }

Reply:
> export interface AppRootProps<T = KeyValue> {

This is the props for the root (and only) page that an app has control of.  It is `/a/${pluginId}/*`